docs/en/connector/sink/Jdbc.mdx

```diff
@@ -56,7 +56,7 @@ user password
 
 ### dbTable [string]
 
-Source data table name
+sink source table name, which will be created if not existed
```

Comment:
Sink table name, not sink source table name, this is ambiguous. The best description is that `Sink table name, if this table is not existed, it wiil be created`.

Reply:
done